src/features/game/components/Decorations.tsx

```diff
@@ -155,8 +155,8 @@ export const Decorations: React.FC = () => {
       {state.inventory["Farm Cat"] && (
         <img
           style={{
-            width: `${GRID_WIDTH_PX * 1}px`,
-            right: `${GRID_WIDTH_PX * 39.78}px`,
+            width: `${GRID_WIDTH_PX * 1.5}px`,
+            right: `${GRID_WIDTH_PX * 39.55}px`,
             top: `${GRID_WIDTH_PX * 28.2}px`,
           }}
```

Comment:
Do we see value in adopting something like `Emotion` where we use the `css` prop over `style` prop?

I'm assuming this will jam inline styles onto the element than using a className to apply the styles.

#performance.

Reply:
@michaeltrilford I'm open to something like this. If it improves the developer experience and the Typescript/CSS intellisense still works 🙏 

The CSS needs an overhaul to get some standards in - especially around the grid positioning 🤔 

It would be great to see a proposal for this 😄 

For now, we will consider it out of scope for this PR

Reply:
Yeh, works great with Typescript. Certainly, I only brought it up to see if we have considered that.

Maybe look into it and let me know. Once installed, I will be able to help convert styles to Emotion. Is there a Storybook?

Re: CSS overhaul 
Point me to some areas that could do with some love.